debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/SqlUtils.java

```diff
@@ -149,7 +149,7 @@ public static String allMinableLogsQuery(Scn scn, Duration archiveLogRetention,
             sb.append("FROM ").append(LOGFILE_VIEW).append(" F, ").append(LOG_VIEW).append(" L ");
             sb.append("LEFT JOIN ").append(ARCHIVED_LOG_VIEW).append(" A ");
             sb.append("ON A.FIRST_CHANGE# = L.FIRST_CHANGE# AND A.NEXT_CHANGE# = L.NEXT_CHANGE# ");
-            sb.append("WHERE A.FIRST_CHANGE# IS NULL ");
+            sb.append("WHERE (A.STATUS <> 'A' OR A.FIRST_CHANGE# IS NULL) ");
```

Comment:
I don't quite understand this predicate change.  The goal here is that we would obtain a list of logs that are to be part of the mining session that are redo logs that have not yet been moved to archive logs.  Did you observe some race or corner condition where not having `STATUS<>"A"` was bad?

Reply:
Please disregard, I missed your explanation of this change in your PR comments.